backend/src/wamp.ts

```diff
@@ -549,6 +458,7 @@ const wampPublish = async <T extends SubscriptionTopicType>(
   namedArgs: SubscriptionTopicTypes[T],
   getSession: () => Promise<autobahn.Session>
 ): Promise<void> => {
+<<<<<<< HEAD
```

Comment:
@luixo please fix it

Reply:
My bad, fixed it.
It's too early to review this, actually :)